docs/modules/operation/pages/deep-dive/meta-data.adoc

```diff
@@ -258,6 +258,7 @@ The following {page-component-title} components allow the use of the Metadata DS
 * <<deep-dive/thresholds/thresholding.adoc#ga-threshold-metadata, Expression-Based Thresholds>>.
 * <<deep-dive/notifications/introduction.adoc#ga-notifications, Notifications>>.
 * <<deep-dive/alarms/alarm-handling.adoc#ga-advanced-alarm-handling, Alarmd Drools rules>>.
+* <<deep-dive/admin/configuration/system-properties.adoc#system-properties, System Properties>>.
```

Comment:
Looks like you missed adding actual doc.

Reply:
This is an enumeration of places that support meta-data. I just added the link to our existing system properties description. So, I did not add any doc-file.